src/core/core.element.js

```diff
@@ -56,12 +56,13 @@ class Element {
 
 	constructor(configuration) {
 		helpers.extend(this, configuration);
+
+		// this.hidden = false; we assume Element has an attribute called hidden, but do not initialize to save memory
+
 		this.initialize.apply(this, arguments);
 	}
 
-	initialize() {
-		this.hidden = false;
-	}
+	initialize() {}
```

Comment:
Agreed. I had actually tried before, but couldn't figure out how to make it work. I'm not sure what was going wrong, but feel free to take a stab in case you have any better luck